src/components/dls/QuranWord/GlyphWord.tsx

```diff
@@ -51,22 +50,23 @@ const GlyphWord = ({
   font,
   isFontLoaded,
 }: UthmaniWordTextProps) => {
-  const quranReaderStyles = useSelector(selectQuranReaderStyles, shallowEqual);
-  const { quranTextFontScale, mushafLines } = quranReaderStyles;
+  const { quranTextFontScale, mushafLines } = useSelector(selectQuranReaderStyles, shallowEqual);
+  const innerHTML = {
+    __html: ` ${getWordText(qpcUthmaniHafs, textCodeV1, textCodeV2, font, isFontLoaded)}`,
+  };
+  const className = classNames(styles.styledWord, {
+    [styles.wordSpacing]: quranTextFontScale < 6,
+    [styles.fallbackText]: !isFontLoaded,
+    [styles[getFontClassName(FALLBACK_FONT, quranTextFontScale, mushafLines, true)]]: !isFontLoaded,
+  });
```

Comment:
I understand this might be subjective and while I feel that assigning a value in a separate `const` instead of inline is clearer but if it's only used once, I recommend we should keep it inline as assigning a value in a const will add some work for the garbage collector to collect that const after it's not longer used.

Reply:
Sure, would that affect the performance?